SU2_CFD/src/numerics/flow/convection/centered.cpp

```diff
@@ -97,7 +97,17 @@ CNumerics::ResidualType<> CCentLaxInc_Flow::ComputeResidual(const CConfig* confi
   DensityInc_i  = V_i[nDim+2];        DensityInc_j  = V_j[nDim+2];
   BetaInc2_i    = V_i[nDim+3];        BetaInc2_j    = V_j[nDim+3];
   Cp_i          = V_i[nDim+7];        Cp_j          = V_j[nDim+7];
-  Enthalpy_i    = Cp_i*Temperature_i; Enthalpy_j    = Cp_j*Temperature_j;
+  if (energy_multicomponent) {
+    Enthalpy_i = V_i[nDim + 9];
+    Enthalpy_j = V_j[nDim + 9];
+    WorkingVariable_i = Enthalpy_i;
+    WorkingVariable_j = Enthalpy_j;
```

Comment:
Don't make these intermediate variables members of the numerics class if you can just use local variables in ComputeResidual